plugins/volumeFilter.go

```diff
@@ -102,8 +105,28 @@ var _ SubmitFilter = &volumeFilter{}
 
 // Validate ensures validity
 func (c *VolumeFilterConfig) Validate() error {
-	if c.isEmpty() {
-		return fmt.Errorf("the volumeFilterConfig was empty")
+	if c.BaseAssetCapInBaseUnits != nil && c.BaseAssetCapInQuoteUnits != nil {
+		return fmt.Errorf("invalid asset caps: only one asset cap can be non-nil, but both are non-nil")
+	}
+
+	if c.BaseAssetCapInBaseUnits == nil && c.BaseAssetCapInQuoteUnits == nil {
+		return fmt.Errorf("invalid asset caps: only one asset cap can be non-nil, but both are nil")
+	}
+
+	if _, e := parseVolumeFilterMode(string(c.mode)); e != nil {
+		return fmt.Errorf("could not parse mode: %s", e)
+	}
+
+	if _, e := queries.ParseDailyVolumeAction(string(c.action)); e != nil {
+		return fmt.Errorf("could not parse action: %s", e)
+	}
+
+	if c.additionalMarketIDs == nil {
```

Comment:
remove these `nil` errors. I think this is what is causing all the issues. these are not required since `nil` is a valid value for both arrays.

Reply:
Got it, this makes sense and completely resolves the issue.